src/wiz.c

```diff
@@ -313,6 +313,8 @@ do_teleport(int descr, dbref player, const char *arg1, const char *arg2)
  * Two kinds of wildcards are understood; * and **.  ** is a recursive
  * wildcard, * is just a 'local' match.
  *
+ * The caller is responsible for permission checks.
```

Comment:
Is this string update also in the header file?

Reply:
That's a valid concern, though I don't think there's a prototype anywhere.

Reply:
You're right, I missed that this is a static due to how git hub clips the file.  Derp!